doc/lua_api.txt

```diff
@@ -6717,7 +6717,7 @@ object you are working with still exists.
         * Fourth column: subject looking to the right
         * Fifth column:  subject viewed from above
         * Sixth column:  subject viewed from below
-* `get_entity_name()` (**Deprecated**: Will be removed in a future version)
+* `get_entity_name()` (**Deprecated**: Will be removed in a future version, use the field `entity.name` instead)
```

Comment:
Might this be more precise? `self` is what the modders deal with and stands for an object reference in general.
```suggestion
* `get_entity_name()` (**Deprecated**: Will be removed in a future version, use the field `self.name` instead)
```

Reply:
`self` stands for the *Lua entity*, not for *object reference*, but yes, it's probably better to use `self` here as it will usually be called `self` in the respective `on_*` methods.